other/docker/windows/build_dependencies.sh

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env sh
 
-set -e -x
+# disable on Cygwin otherwise some builds fail
```

Comment:
What's the error you are getting?

Reply:
`autogen.sh` failing for libsodium (like for opus).

Perhaps I could build libsodium from tarball like Opus than I could propably enable back `set -e -x` thing.